js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let midImg = document.getElementById("cta-img");
+midImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let lastImg = document.getElementById("middle-img");
+lastImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+const navBar = document.querySelector("nav").children;
+navBar[0].textContent = siteContent["nav"]["nav-item-1"];
+navBar[1].textContent = siteContent["nav"]["nav-item-2"];
+navBar[2].textContent = siteContent["nav"]["nav-item-3"];
+navBar[3].textContent = siteContent["nav"]["nav-item-4"];
+navBar[4].textContent = siteContent["nav"]["nav-item-5"];
+navBar[5].textContent = siteContent["nav"]["nav-item-6"];
```

Comment:
you can accomplish this nicely with a for loop, to loop through all the children in nav.  

![image](https://user-images.githubusercontent.com/30554629/59724131-6f79ad80-91ee-11e9-94d6-88958cdfc1c8.png)